templates/OperationSendRequestSnippet.stencil

```diff
@@ -15,6 +15,7 @@ let context = PipelineContext.of(keyValues: [
    {% endfor %}
 ])
 context.add(cancellationToken: options?.cancellationToken, applying: self.options)
+context.merge(with: options?.context)
```

Comment:
The method `context.merge` sounds similar to `context.add`.  Especially if one calls context.add and next with context.merge.

Reply:
Add only adds one thing. Merge combines all values from another context into the first one (in this case, any values the user has created)